clippy_lints/src/utils/mod.rs

```diff
@@ -75,8 +75,8 @@ pub fn parse_msrv(msrv: &str, sess: Option<&Session>, span: Option<Span>) -> Opt
     None
 }
 
-pub fn meets_msrv(msrv: Option<&VersionReq>, lint_msrv: &Version) -> bool {
-    msrv.map_or(true, |msrv| !msrv.matches(lint_msrv))
+pub fn meets_msrv(msrv: Option<&RustcVersion>, lint_msrv: &RustcVersion) -> bool {
+    msrv.map_or(true, |msrv| msrv >= lint_msrv)
```

Comment:
@flip1995 should we add a test to check equality of `msrv` as well?

Reply:
Yeah a test for equality and for just in and just out would be good. You can put them in the same test file by using `mod { #![msrv] }`